apps/cowswap-frontend/src/modules/limitOrders/services/safeBundleFlow/index.ts

```diff
@@ -42,12 +40,7 @@ export async function safeBundleFlow(
   const { account, recipientAddressOrName, sellToken, buyToken, inputAmount, outputAmount, isSafeWallet } =
     params.postOrderParams
 
-  // TODO: remove once we figure out what's adding this to appData in the first place
```

Comment:
@alfetopito I've deleted this check because now we may have other hooks besided permit. I hope it's ok